docs/sources/datasources/pyroscope/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+aliases:
+  - ../features/datasources/phlare/
+  - ../features/datasources/grafana-pyroscope/
+  - ../datasources/grafana-pyroscope/
+description: Horizontally-scalable, highly-available, multi-tenant continuous profiling
+  aggregation system. OSS profiling solution from Grafana Labs.
+keywords:
+  - grafana
+  - phlare
+  - guide
+  - profiling
+  - pyroscope
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Grafana Pyroscope
+weight: 1150
+---
+
+# Grafana Pyroscope data source
+
+Grafana Pyroscope is a horizontally scalable, highly available, multi-tenant, OSS, continuous profiling aggregation system. Add it as a data source, and you are ready to query your profiles in [Explore][explore].
+
+To learn more about profiling and Pyroscope, refer to the [Introduction to Pyroscope](/docs/pyroscope/introduction/).
+
+For information on configuring the Pyroscope data source, refer to the [Configure the Grafana Pyroscope data source](./configure-pyroscope-data-source) documentation.
```

Comment:
I think _the X documentation_ is only needed when you are linking from outside of documentation pages into them.

https://grafana.com/docs/writers-toolkit/write/links/useful-link-text/

```suggestion
For information on configuring the Pyroscope data source, refer to [Configure the Grafana Pyroscope data source](./configure-pyroscope-data-source).
```